lib/memcached.js

```diff
@@ -112,8 +111,7 @@ Client.config = {
   nMemcached.prototype.__proto__ = require('events').EventEmitter.prototype;
 
   var memcached = nMemcached.prototype
-    , privates = {}
-    , undefined;
```

Comment:
sure, but re-defining a scoped `undefined` variable actually makes checking against `undefined` safe as you can override the global `undefined` variable with things.